### PR TITLE
udpate

### PR DESCRIPTION
**Pull request by zzhang2019**

### CODE REVIEW

**Comment by zzhang2019** [comment]
test

**Comment by zzhang2019** [comment]
test

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-AIRFLOW_VERSION ?= 1.8.0.0
-KUBECTL_VERSION ?= 1.6.1
-KUBE_AIRFLOW_VERSION ?= 0.10
+AIRFLOW_VERSION ?= 1.10.2.0
+KUBECTL_VERSION ?= 1.14.1
+KUBE_AIRFLOW_VERSION ?= 0.18
 
-REPOSITORY ?= mumoshu/kube-airflow
+REPOSITORY ?= littlezerg/kube-airflow
```
Comment (gsemet) [inline Makefile:5]: Not sure if this wanted

File: script/entrypoint.sh [hunk 2]
```diff
@@ -10,7 +10,8 @@ RABBITMQ_CREDS="${RABBITMQ_CREDS:-airflow:airflow}"
 RABBITMQ_MANAGEMENT_PORT=15672
 FLOWER_URL_PREFIX="${FLOWER_URL_PREFIX:-}"
 AIRFLOW_URL_PREFIX="${AIRFLOW_URL_PREFIX:-}"
-LOAD_DAGS_EXAMPLES="${LOAD_DAGS_EXAMPLES:-true}"
+# LOAD_DAGS_EXAMPLES="${LOAD_DAGS_EXAMPLES:-true}"
+LOAD_DAGS_EXAMPLES=False
```
Comment (gsemet) [inline script/entrypoint.sh:14]: Why that ?

File: Makefile (hunk 1 above)
Comment (zzhang2019) [reply Makefile:5]: of course not, it need to be cleaned

**Pull request closed without merging**